pkgs/applications/misc/glava/default.nix

```diff
@@ -22,12 +22,14 @@ let
 in
   stdenv.mkDerivation rec {
     name = "glava-${version}";
-    version = "1.5.5";
+    version = "1.5.8";
 
-    src = fetchgit {
-      url = "https://github.com/wacossusca34/glava.git";
+    src = fetchFromGitHub {
+      owner = "wacossusca34";
+      repo = "glava";
       rev = "v${version}";
-      sha256 = "0mpbgllwz45wkax6pgvnh1pz2q4yvbzq2l8z8kff13wrsdvl8lh0";
+      sha256 = "1n3m0zvdmprj3iwgbdxsnfzhc74xhhmmgb2hl34fbnzrnxc6iccw";
+      fetchSubmodules = true;
```

Comment:
Why are the submodules needed? Could you leave a comment with an explanation.

Reply:
Sorry about that, just reviewed it again and it turns out submodules aren't needed.